tests/permissions/test_permissions.py

```diff
@@ -114,3 +116,16 @@ def test_write_role():
         )
 
     ln.ULabel(name="new label default space").save()
+
+
+# below is an integration test that should run last
+def test_lamin_dev():
+    script_path = Path(__file__).parent.resolve() / "scripts/example_script.py"
+    result = subprocess.run(  # noqa: S602
+        f"python {script_path}",
+        shell=True,
+        capture_output=True,
+    )
+    print(result.stdout.decode())
+    print(result.stderr.decode())
+    assert result.returncode == 0
```

Comment:
```suggestion
    result = subprocess.run(  # noqa: S602
        f"python {script_path}",
        shell=True,
        check=True,
    )
```

Reply:
Without these print statements an error is impossible to debug. So, I would much prefer to leave them in the code. Because this test depends on the state of lamin-dev it's not so unlikely that it breaks and then it will save us time if we know why it broke.

Reply:
`check=True` reports the traceback if an error happens.

Reply:
And the stdout?

Reply:
But sure that's way more elegant, we wouldn't even need to define the result even though I'm often checking for certain logging messages (not here though)

Reply:
>If check is true, and the process exits with a non-zero exit code, a [CalledProcessError](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError) exception will be raised. Attributes of that exception hold the arguments, the exit code, and stdout and stderr if they were captured.

Reply:
Got it!